src/qml/patchmanagermodel.cpp

```diff
@@ -239,6 +239,17 @@ QString PatchManagerModel::patchName(const QString &patch) const
     return m_patchMap[patch]->details()->value(QStringLiteral("display_name")).toString();
 }
 
+bool PatchManagerModel::isApplied(const QString &name) const
+{
+    // FIXME: there certainly is a more efficient way, e.g. std::find_if?
```

Comment:
If this was sorted, std::lower_bound :)
Would've been cool if there was already a map populated onPatchedChanged from JS directly - not sure if this is triggered inside the PatchObject::setData call.

(also that loop in PatchObject::setData is very weird. I almost filed a bug, it is like a tech debt, not actually a bug)

Anyway, I'm fine with this FIXME for the moment -> meaning that we should still find a better solution.